### ISSUE TITLE
Unable to install or update packages: "apt-get: not found"

### ISSUE THREAD

**Issue by bee-rock** [offensive-security/kali-nethunter#988]
### Device:

Nexus 10, 5.1.1 (LMY49J)

### OS version (KitKat/Lollipop/Marshmallow/Nougat):

Lollipop

### Built from repo (date and build command) or downloaded from website (links):

python build.py -d manta --lollipop --rootfs full --nightly
Sat Sep 16 14:03:31 MDT 2017
nethunter-manta-lollipop-kalifs-full-20170916_125935.zip

### Output of `cat /proc/version` in adb shell or NetHunter Terminal:

Linux version 3.4.67-00092-g7b02438 (root@kali) (gcc version 4.7 (GCC) ) #1 SMP PREEMPT Thu Sep 24 01:04:17 UTC 2015

## Issue:

I cannot update any packages:

Open Nethunter -> Click Kali Chroot Manager -> Add Metapackages -> Kali Linux -> Install & Update

#1 u0_a81@manta:/ $ apt-get
/system/bin/sh: apt-get: not found
#1 u0_a81@manta:/ $ su
root@manta:/ # apt-get
tmp-mksh: apt-get: not found

I used https://github.com/offensive-security/nethunter-LRT, first with stockNexusFlash.sh and then twrpFlash.sh with the following images:

SuperSU: SR1-SuperSU-v2.82-SR1-20170608224931.zip
TWRP: twrp-3.1.1-0-manta.img
Stock image: mantaray-lmy49j-factory-b24f40e6.zip

I also noticed that the Busybox Version says:
Busybox not detected!

I tried using the official stable release, nethunter-manta-lollipop-3.0.zip, and encountered the same problem and none of the kali applications worked and said "Unfortunately, (eg., Nethunter, Nethunter Terminal, androidVNC, etc..) has stopped working".

Please let me know if there is any additional information I can add.

**Comment by jcadduono** [comment]
you are not in the rootfs, need to use `su 0 bootkali_login`

**Comment by bee-rock** [comment]
@jcadduono Thanks for the quick response. Please excuse my ignorance ahead of time, but is there a way to make this persistent? i.e,. So that the Nethunter application uses rootfs when attempting to update? Or is this a sign that there is something else wrong?

**Comment by bee-rock** [comment]
After running the following commands: apt-get update, apt-get upgrade, apt-get dist-upgrade, apt-get install kali-linux, and restarting, I was able to use the Nethunter GUI to select packages to update. Something quirky on the fresh flash though.

**Comment by jcadduono** [comment]
It is probably because the su 0 bootkali_login put selinux into permissive which then allows the app to function

**Comment by bee-rock** [comment]
I didn't know that was possible if SELinux is disabled?

root@kali:~# sestatus
SELinux status:           disabled

Upon restarting the tablet, the problem persists and "su 0 bootkali_login" doesn't seem to affect it.

**Comment by bee-rock** [comment]
Grasping at straws I tried a few things I found on other threads:
https://github.com/offensive-security/kali-nethunter/issues/329

"I went to Kali Services and turned on SSH then all (and by that I mean opening Kali Terminal and NMAP) was good."

Aha, I tried the same thing. I can reliably restart the tablet and nothing works (no commands work, can't identify BusyBox or any of the Network Interfaces, etc), but enabled SSH, or any of the kali services, seems to fix things. This is why it started working for me earlier, not the "su 0 bootkali_login" command, but rather when I was tinkering with things I enabled Metasploit. 

Let me know if there are any logs I can provide, it's a bizarre side effect.
